addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -122,6 +123,11 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
             errorMessages.push(_t("The booth category doesn't exist."));
         }
 
+        if (errors.includes('existingPartnerError')) {
+            errorMessages.push(_t("Looks like your email is linked to an existing account."));
```

Comment:
Would make complete sentences
```suggestion
            errorMessages.push(_t("It looks like your email is linked to an existing account."));
```